src/main/java/com/exasol/adapter/properties/ValidatorChain.java

```diff
@@ -0,0 +1,37 @@
+package com.exasol.adapter.properties;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import com.exasol.adapter.AdapterProperties;
+
+/**
+ * Validate the specified properties by applying all property validators subsequently.
+ */
+public class ValidatorChain {
```

Comment:
```suggestion
public class ValidatorChain implements PropertyValidator {
```
Would be nice if possible

Reply:
👍